README.md

```diff
@@ -167,7 +167,7 @@ npm run test
 
 ## LICENSE
 
-Copyright 2017 Cerner Innovation, Inc.
+Copyright 2018 Cerner Innovation, Inc.
```

Comment:
Should this be an inclusive date? 2017-2018?

Reply:
Also, we likely need to update the Copyright date of each component's readme as well

Reply:
@emilyrohrbough We should check with legal about the date. We may be able to go with something like `2017 - Present`

Reply:
We'll want to update this to `2017 - 2018`. The Copyright years listed here match [Apache Foundation's recommendation for year(s) of distribution of the current and past versions of the product](https://www.apache.org/legal/src-headers.html#notice)